css/style.css

```diff
@@ -342,3 +345,143 @@ hr {
   background-color: #6070ff;
   color: #fff;
 }
+
+@media only screen and (min-width: 175px) and (max-width: 420px) and (orientation: portrait) {
+  body {
+    background-image: url("../img/header-background-mobile.png");
+    background-repeat: no-repeat;
+    background-position: 0 0 0 0;
+  }
+
+  .nav-container {
+    display: none;
+  }
+
+  .nav-space {
+    display: none;
+  }
+}
+
+@media (min-width: 421px) {
```

Comment:
- [ ] Kindly work with just one media query, it is 768px, for any device equal to or greater than this size CSS is going to apply the desktop version styles.